_layouts/guides.html

```diff
@@ -59,7 +59,7 @@ <h1 class="text-caps">{{page.title}} {{page.docversion}}</h1>
       {{ content }}
     </div>
     <div class="grid__item width-4-12 width-12-12-m tocwrapper">
-      <div class="hide-mobile toc">{{ page.document | tocify_asciidoc: 2 }}</div>
```

Comment:
So the `: 2` is used to specify the default value, or? I mean if we remove it what's the default?

Reply:
Having a value here enforces the value and completely ignore the `:toclevels:` at the guide level.

The default for `:toclevels:` is 2 (https://docs.asciidoctor.org/asciidoc/latest/toc/levels/#set-toclevels) so by removing this, we don't change the default behavior and we allow to override it at the guide level (as I did for Qute).